src/Marian/index.ts

```diff
@@ -238,7 +238,7 @@ export default class Marian {
     res.end(responseBody);
   }
 
-  private async fetchResults(parsedUrl: URL): Promise<Document[]> {
+  private async fetchResults(parsedUrl: URL, req: http.IncomingHttpHeaders): Promise<Document[]> {
```

Comment:
Can we rename `req` to `reqHeaders` for consistency with its type? Thought this was the request object itself at first